app/controllers/merchant/items_controller.rb

```diff
@@ -5,6 +5,23 @@ def index
     @merchant_employee = User.find(current_user.id)
   end
 
+  def update
+    @item = Item.find(params[:item_id])
```

Comment:
Where is the request that is being routed to this action?
Is this `@item` being called in another view page?

Reply:
A future user story will be using the update route, so I needed something else. I wrote this code initially, but didn't delete it before sending the PR.

Reply:
Sounds good - I'm not sure what other approach to take here.